lib/infrastructure/models/user_model.dart

```diff
@@ -113,6 +113,7 @@ class UserModel extends Equatable {
 
   UserModel({
     required this.id,
+    this.url = '',
```

Comment:
are you changing the db model?

Reply:
Yes, we need to save the deep link URL that's why.